File: src/main/java/pe/com/test_qsystem/distribution_center/impl/TrujilloDistributionCenterImpl.java
package pe.com.test_qsystem.distribution_center.impl;

import pe.com.test_qsystem.distribution_center.DistributionCenter;

public class TrujilloDistributionCenterImpl implements DistributionCenter {

    @Override
    public boolean support(String center) {
        return center.equalsIgnoreCase("TRUJILLO");
    }

    @Override
    public void operate(String center) {
        System.out.println("Operate in Trujillo distribution center.");
    }
    
}

File: src/main/kotlin/pe/com/qsystem/test_qsystem/distribution_center/impl/TrujilloDistributionCenterImpl.kt
package pe.com.qsystem.test_qsystem.distribution_center.impl

import org.slf4j.LoggerFactory
import pe.com.qsystem.test_qsystem.distribution_center.DistributionCenter

class TrujilloDistributionCenterImpl : DistributionCenter {
        
    private val log = LoggerFactory.getLogger(javaClass)

    override fun support(center: String) = center.contentEquals("TRUJILLO", true)

    override fun operate(center: String) {
        log.info("Operate in Trujillo distribution center.")
    }
}